docs/source/example_overview.md

```diff
@@ -0,0 +1,73 @@
+# Examples
+
+
+## Introduction
+
+The examples should work in any of the following settings (with the same script):
+   - single CPU or single GPU
```

Comment:
I don't think the examples can actually run on CPU right? E.g. if we activate 8-bit, the PPO example likely won't work IIRC
```suggestion
   - single GPU
```